backend/module-campaigns/src/Podkrepibg.Campaigns.Application/Beneficiaries/Commands/CreateBeneficiary/CreateBeneficiaryValidator.cs

```diff
@@ -0,0 +1,31 @@
+namespace Podkrepibg.Campaigns.Application.Beneficiaries.Commands.CreateBeneficiary
+{
+    using System;
+    using FluentValidation;
+
+    public class CreateBeneficiaryValidator : AbstractValidator<CreateBeneficiaryCommand>
+    {
+        public CreateBeneficiaryValidator()
+        {
+            RuleFor(b => b.Request.FirstName)
+                .NotEmpty();
+
+            RuleFor(b => b.Request.LastName)
```

Comment:
Should we not use MaximumLength to validate whether the last name is longer than the database column? If we decide to do it/use it we should probably do that for all of the string values.